docs/changelog.md

```diff
@@ -10,18 +10,39 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes
 
 - Python dependencies are set up locally in `.venv` environment. Therefore, you should make the following changes: 
-  - fix `docker-compose.yml` file to run `healthcheck.py` for sc-server in `.venv` or use `scripts/healthcheck.sh` 
-encapsulated this logic;
-  - and use `.venv` to launch your python scripts if they use sc-machine python dependencies.
-- `sc-server` binary and `run_sc_server.sh` script were removed. Use `sc-machine` binary instead.
+    - fix `docker-compose.yml` file to run `healthcheck.py` for sc-server in `.venv` or use `scripts/healthcheck.sh` encapsulated this logic;
+    - and use `.venv` to launch your python scripts if they use sc-machine python dependencies.
+- Build system of the sc-machine was upgraded:
+    - The way to manage build dependencies was changed to [Conan](https://conan.io). You can install sc-machine and its dependencies with Conan. You don't have to worry about installing sc-machine dependencies on your OS anymore. See how to do it -- [Build System](build/build_system.md).
+    - CMake presets should be used to build the sc-machine. They are located in `CMakePresets.json`.
+    - Location of the sc-machine build was changed. The following new locations are:
+        - build tree -- `build/<Debug|Release>`, 
+        - binaries -- `build/<Debug|Release>/bin`, 
+        - libraries -- `build/<Debug|Release>/lib`,
+        - extensions -- `build/<Debug|Release>/lib/extensions`.
+    - Using sc-machine as a library is much more optimized: adding it to the CMake tree is no longer necessary. You can install sc-machine packages and import sc-machine targets into your cmake using `find_package(sc-machine REQUIRED)`. But if you include sc-machine into CMake tree of your project, remember that the minimum version of cmake for the machine is 3.24.
```

Comment:
Actually it's not the case. While building sc-machine itself requires CMake 3.24 because we use presets (3.21 because of `conan_provider.cmake` as well), none of this gets exported to the library consumers, so they can use basically any *modern enough* version of CMake